tests/mock/Concrete/PlatformOrderDecorator.php

```diff
@@ -0,0 +1,899 @@
+            "{$prefix}_buyer_street_title{$index}" => "street",
+            "{$prefix}_buyer_street_number{$index}" => "number",
+            "{$prefix}_buyer_neighborhood{$index}" => "neighborhood",
+            "{$prefix}_buyer_street_complement{$index}" => "complement",
+            "{$prefix}_buyer_city{$index}" => "city",
+            "{$prefix}_buyer_state{$index}" => "state",
+            "{$prefix}_buyer_zipcode{$index}" => "zipCode",
+            "{$prefix}_buyer_home_phone{$index}" => "homePhone",
+            "{$prefix}_buyer_mobile_phone{$index}" => "mobilePhone"
+        ];
+
+        $multibuyer = new \stdClass();
+
+        foreach ($fields as $key => $attribute) {
+            $value = $additionalInformation[$key];
+
+            if ($attribute === 'document' || $attribute === 'zipCode') {
+                $value = preg_replace(
+                    '/\D/',
+                    '',
+                    $value
+                );
+            }
+
+            $multibuyer->$attribute = $value;
+        }
+
+        return $multibuyer;
+    }
+
+    private function extractPaymentDataFromMundipaggBilletCreditcard(
+        $additionalInformation,
+        &$paymentData, $payment
+    )
+    {
+        $moneyService = new MoneyService();
+        $identifier = null;
+        $customerId = null;
+
+        $brand = null;
+        try {
+            $brand = strtolower($additionalInformation['cc_type']);
+        } catch (\Throwable $e) {
+
+        }
+
+        if (isset($additionalInformation['cc_token_credit_card'])) {
+            $identifier = $additionalInformation['cc_token_credit_card'];
+        }
+
+        if (
+            !empty($additionalInformation['cc_saved_card']) &&
+            $additionalInformation['cc_saved_card'] !== null
+        ) {
+            $identifier = null;
+        }
+
+        if ($identifier === null) {
+            $objectManager = ObjectManager::getInstance();
+            $cardRepo = $objectManager->get(CardsRepository::class);
+            $cardId = $additionalInformation['cc_saved_card'];
+            $card = $cardRepo->getById($cardId);
+
+            $identifier = $card->getCardToken();
+            $customerId = $card->getCardId();
+        }
+
+        $newPaymentData = new \stdClass();
+        $newPaymentData->identifier = $identifier;
+        $newPaymentData->customerId = $customerId;
+        $newPaymentData->brand = $brand;
+        $newPaymentData->installments = $additionalInformation['cc_installments'];
+
+        $newPaymentData->saveOnSuccess =
+            isset($additionalInformation["cc_savecard"]) &&
+            $additionalInformation["cc_savecard"] === '1';
+
+        $amount = str_replace(
+            ['.', ','],
+            "",
+            $additionalInformation["cc_cc_amount"]
+        );
+        $newPaymentData->amount = $moneyService->floatToCents($amount / 100);
+
+        $creditCardDataIndex = AbstractCreditCardPayment::getBaseCode();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Mundipagg\Core\Payment\Aggregates\Payments\AbstractCreditCardPayment' in method 'extractPaymentDataFromMundipaggBilletCreditcard'.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4926694)